lib/system/database/definitions/mysql.rb

```diff
@@ -204,6 +204,12 @@
     SQL
   end
 
+  trigger 'policies' do
```

Comment:
We should open an issue in https://github.com/3scale/infrastructure to recreate triggers.